src/libraries/System.Collections.Immutable/tests/ImmutableSortedSetTest.nonnetstandard.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Collections.Immutable.Tests
     public partial class ImmutableSortedSetTest : ImmutableSetTest
     {
         [Fact]
-        [ActiveIssue("Sporadic failure, needs a port of https://github.com/dotnet/coreclr/pull/4340", TargetFrameworkMonikers.NetFramework)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Sporadic failure, needs a port of https://github.com/dotnet/coreclr/pull/4340")]
```

Comment:
I thought about filing an issue for this but given that https://github.com/dotnet/coreclr/pull/4340 is four years old now and wasn't ported to .NET Framework we can probably assume it won't happen.